### PR TITLE
Added basscss

### PR DESCRIPTION
**Pull request by nickbristow**
Added styling to index
  Added success page
  Added styling for success bar
  Added login.gov logo

### CODE REVIEW

**Comment by nickbristow** [comment]
identity

File: views/success.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<%= erb :user_info, locals: { userinfo: userinfo} %>
```
Comment (zachmargolis) [inline views/success.erb]: nit: space before the `}`, should be `userinfo }`

File: views/user_info.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<div class="p2 bg-green white clearfix">
+  <% if userinfo %>
+    <form action="/auth/saml/logout" method="POST" class="right sm-show">
+      <button type="submit" class="btn p0 h6 white">Log out</button>
+    </form>
+  <% end %>
+  <span class="bold">Success!</span>
+  <% if userinfo %>
+    <span>Email: <strong><%= userinfo[:email] %></strong></span>
+    <% unless userinfo[:first_name].nil? %>
```
Comment (zachmargolis) [inline views/user_info.erb]: I would just do `if userinfo[:first_name]` rather than `unless ....nil?`

File: views/user_info.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      <span class="ml1 mr1">|</span>
+      <span>Name: <strong><%= userinfo[:first_name] %> <%= userinfo[:last_name] %></strong></span>
+    <% end %>
+    <% unless userinfo[:social_security_number].nil? %>
```
Comment (zachmargolis) [inline views/user_info.erb]: ditto for `if` rather than `unless ...nil?`

**Pull request merged**